addons/website_forum/controllers/website_forum.py

```diff
@@ -690,12 +690,19 @@ def _prepare_open_forum_user(self, user, forums, **kwargs):
         Activity = request.env['mail.message']
         Followers = request.env['mail.followers']
         Data = request.env["ir.model.data"]
+        search_values = {}
 
         # questions and answers by user
-        user_question_ids = Post.search([
+        questions_domain = [
             ('parent_id', '=', False),
-            ('forum_id', 'in', forums.ids), ('create_uid', '=', user.id)],
-            order='create_date desc')
+            ('forum_id', 'in', forums.ids), ('create_uid', '=', user.id)]
+        questions_domain_without_search = questions_domain
```

Comment:
Maybe we could name "base_question_domain" and "search_question_domain", something like that ?